Makefile

```diff
@@ -1,21 +1,49 @@
 # Watson
 
+PYTHON ?= python
+PIP ?= pip
+
+VENV = virtualenv
+VENV_ARGS = -p $(PYTHON)
+VENV_DIR = $(CURDIR)/.env
```

Comment:
I think another name should be chosen for the venv dir (`.venv`?) since `.env` conflicts with the [`autoenv`](https://github.com/kennethreitz/autoenv) file